README.md

```diff
@@ -27,6 +27,7 @@ USAGE
     [--dev eth1...] \
     [--multicast 224.0.0.251] \
     [-s <spoof_source_ip>]
+    [-t <overridden_target_ip>]
```

Comment:
It also needs a backslash (`\`) on the previous line to show that the command is continued.

Reply:
Fixed, hopefully.